.github/workflows/knative-style.yaml

```diff
@@ -60,3 +60,13 @@ jobs:
               exit 1
           fi
           echo "${{ github.repository }} is formatted correctly."
+      
+      - name: Language
+        shell: bash
+        run: |
+          changed=$(git diff --name-only HEAD HEAD~1 | grep -v '/vendor\|/third-party\|.github\|go.sum\|go.mod\|.md')
+          if [ ! -z "$changed" ]; then
+            grep -ni 'whitelist\|blacklist\|master\|slave' $changed | grep -v '/master/'
```

Comment:
I'm not sure this fully works; consider the following addition:

```
+// Block the blacklist which is stored in https://github.com/knative/serving/master/blob/....
```

I think this will cause the line to be matched by the `grep -v`, causing the entire expression to be interpreted as "false".

One way to do this would be to count matches for each:

```shell
DIVISIVE_COUNT=$(grep -oi "$DIVISIVE_TERMS" | wc -l)
GITHUB_BRANCH_COUNT=$(egrep -oi 'https://github.com/[^ ]+/master/' | wc -l)
if [[ $DIVISIVE_COUNT -gt $GITHUB_BRANCH_COUNT ]]; then
  echo 'Please replace usages with less divisive alternatives, see ....'
  exit 1
if

Reply:
A, possibly simpler, option is to use a regex tool that supports negative look ahead, something like (PCRE here):
  (?!/master/)(.|^)master

Another option is to split the check for master from that from the other terms.

Reply:
No longer relevant with `woke`.